learn/guides/ballerina-shell.md

```diff
@@ -31,23 +34,39 @@ Type /exit to exit and /help to list available commands.
 =$ 
 ```
 
-## Stopping the Shell
+## Available commands
 
-Execute the `/exit` command to exit the Shell.
+Execute the `/help` command to view available commands.
 
 ```bash
-=$ /exit                                                                                                                                                                                                           
-| Bye!!!
+=$ /help                 
+| Ballerina Shell Help
+| 
+| Type a Ballerina language expression, statement, or declaration.
+| Or type one of the following commands:
+| 
+| /exit           - Exit the shell.
+| /debug          - Toggle the debug mode ON/OFF.
+| /reset          - Reset REPL state.
+|                   All previous code snippets will be removed from REPL memory.
+| /file <FILE>    - Load declarations defined in <FILE>. e.g., /file my_file.bal
+| /remove <NAMES> - Delete the declarations specified by <NAMES>. e.g., /remove function1 var1 var2
+| /imports        - List the available imports.
+| /vars           - List the declared variables and their values.
+| /dclns          - List the user defined module level declarations.
+| /help           - Displays the help text.
+| /help <TOPIC>   - Get help on <TOPIC>. eg: /help var
+| /help topics    - See all available topic names.
```

Comment:
```suggestion
| /help topics    - See all the available topic names.
```

Reply:
Updated in 9d7f35613ad026f7312d90a4f2bc24c05d95c1ec